mods/d2k/rules/player.yaml

```diff
@@ -72,4 +72,4 @@ Player:
 	ProvidesTechPrerequisite@all:
 		Name: Unrestricted
 		Prerequisites: techlevel.low, techlevel.medium, techlevel.high, techlevel.superweapons
-
```

Comment:
This newline removal will create unnecessary noise when we run our mod upgrade utility.